src/operator-docs/combination/mergeAll.ts

```diff
@@ -1,6 +1,77 @@
-import { OperatorDoc } from '../operator.model';
+import { OperatorDoc } from "../operator.model";
```

Comment:
Use single quote here. This will be handled automagically once #148 is merged.

Reply:
@xlozinguez It doesn't take effect as pre-commit hook change it to double quotes again. Probably would be done after #148 is merged.